src/components/FiltersDrawer/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import { SideNav, SideNavGroup } from "@leafygreen-ui/side-nav";
+import { Body } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { QueryParams } from "constants/queryParams";
+import { size, zIndex } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import Filter from "./Filter";
+
+interface FiltersDrawerProps {
+  ["data-cy"]?: string;
+}
+
+const FiltersDrawer: React.FC<FiltersDrawerProps> = ({ "data-cy": dataCy }) => {
+  const [collapsed, setCollapsed] = useState(true);
```

Comment:
I could see some benefit to defaulting collapsed to `false`... may help prevent a few "how do I apply filters" questions from users 😆

Reply:
I changed it to use a cookie!